tests/main/uc20-snap-recovery/task.yaml

```diff
@@ -0,0 +1,48 @@
+summary: Integration tests for the snap-recovery binary
+
+# one system is enough, its a very specialized test for now
+systems: [ubuntu-18.04-64]
+
+restore: |
+    if [ -f loop.txt ]; then
+        losetup -d "$(cat loop.txt)"
+    fi
+
+prepare: |
+    echo "Create a fake block device image"
+    dd if=/dev/zero of=fake.img bs=1M seek=2000 count=1
+
+    echo "Setup the image as a block device"
+    losetup -fP fake.img
```

Comment:
The `-P` part seems useless since we just created an empty file.

Reply:
Indeed, I will remove this.